web/wp-content/themes/dctx-v2/assets/bower_components/jquery/src/var/ObjectFunctionString.js

```diff
@@ -0,0 +1,7 @@
+define( [
+	"./fnToString"
+], function( fnToString ) {
+	"use strict";
+
+	return fnToString.call( Object );
+} );
```

Comment:
:no_entry_sign: **Error**: Closing parenthesis of a multi-line function call must be on a line by itself (*PEAR.Functions.FunctionCallSignature.CloseBracketLine*).